src/containers/WishListsPage.jsx

```diff
@@ -81,6 +82,11 @@ class WishListsPage extends React.Component {
       })
         .then(res => res.json())
         .then((res) => {
+          if (res.links.length === 0 && !res._defaultwishlist) {
```

Comment:
The check `res.links.length === 0` is not triggered, so the message never gets displayed